selectMenu/select.rb

```diff
@@ -0,0 +1,25 @@
+def select_option(opt, app)
+  options = {
+    1 => -> { app.add_book },
+    2 => -> { app.add_music_album },
+    3 => -> { app.add_movie },
+    4 => -> { app.add_game },
+    5 => -> { app.list_all_books },
+    6 => -> { app.list_all_music_albums },
+    7 => -> { app.list_all_movies },
+    8 => -> { app.list_all_games },
+    9 => -> { app.list_all_genres },
+    10 => -> { app.list_all_labels },
+    11 => -> { app.list_all_authors },
+    12 => -> { app.list_all_sources },
+    0 => lambda {
+      puts 'Thanks for using this app'
+      exit
+    }
+  }
```

Comment:
[OPTIONAL] Since you are not implementing the movie part, kindly remove options 3, 7 & 12  to avoiding producing an error when a user tries to select them. :wink: